uktrade/data-hub-frontend
# Added additional post filter for is_global_ultimate

## Description of change

Add is_global_ultimate: true to the post body of a company collection search when filtering on Global HQ as we also want to return Ultimate HQ companies in addition.

## Test instructions

Ensure the front end is pointing to the dev API:
When in the companies collection page click on Global HQ. One List Corp should be returned. Click through to the One List Corp overview page. You should see that this is labelled as an Ultimate HQ

## Screenshots

![Screenshot 2023-09-06 at 15 08 04](https://github.com/uktrade/data-hub-frontend/assets/72826129/560f9a2d-9d97-4645-bd14-2ea2a6e9f00c)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
